pom.xml

```diff
@@ -11,7 +11,9 @@
     <url>https://corchestra.ru/wiki/index.php?title=Celesta</url>
 
     <modules>
+        <!--
```

Comment:
Please revert, we still need this module 😉

Reply:
Done, it was commented by error